irods_audit_elk_stack/README.md

```diff
@@ -0,0 +1,106 @@
+# iRODS Audit Demonstration ELK Stack Container
+
+This is an ELK-like stack container used for demonstrating the
+[audit rule engine plugin](https://github.com/irods/irods_rule_engine_plugin_audit_amqp>) for
+[iRODS](https://irods.org/)
+
+## CAVEAT EMPTOR
+
+While some effort has been taken to ensure this container lives up to certain standards of quality,
+it is by no means production-ready.
+This is by design.  
+The express purpose of this container is to demonstrate the audit rule engine plugin,
+not to serve as an example of a properly configured ELK stack for use in production environments.  
+Case in point:
+Elasticstack security is explicitly disabled,
+and the Python script standing in for Logstash was not written with performance or resilience in mind.  
```

Comment:
Probably want to merge some of these lines and let github handle line wrapping.

Reply:
Single line-breaks are omitted when rendering Markdown unless the line ends in two spaces, so most of these go away anyway. These no-op line-breaks are here more for my benefit and I will make it neater once I'm done writing the README.

Reply:
To see how this actually looks when it's rendered, you can look at the file in my repo: https://github.com/SwooshyCueb/irods-contrib/blob/elk-upd.main/irods_audit_elk_stack/README.md

Reply:
Consider adding an empty line after L11 and L13. Creating some separation at those locations makes it easier to read.